aiographite/aiographitesend.py

```diff
@@ -0,0 +1,325 @@
+import asyncio
+from .graphite_escaping import GraphiteEncoder
+import pickle
+import struct
+import socket
+import os
+import time
+from typing import Dict, Tuple, List
+
+
+DEFAULT_GRAPHITE_PLAINTEXT_PORT = 2003
+DEFAULT_GRAPHITE_PICKLE_PORT = 2004
+SUPPORT_PROTOCOLS = ["plaintext", "pickle"]
+
+
+class AioGraphiteSendException(Exception):
+    pass
+
+
+class PlaintextProtocol(object):
+
+	def data_formate_function(self, metric: str, value: int, timestamp: int):
+		"""
+			@return: required data formate when sending data through 'plaintext' protocol
+			@return_type: String
+		"""
+		formatted_data = " ".join([metric, str(value), str(timestamp)])
+		return formatted_data + "\n"
+
+
+	def generate_message_function(self, listOfPlaintext: List[str]):
+		"""
+			return the required message formate for protocol 'plaintext'
+			@param: 
+				listOfPlaintext: ["metric1 value1 timestamp1", "metric2 value2 timestamp2", ...]
+				type: List of String
+		"""
+		return "".join(listOfPlaintext).encode('ascii')
+		
+
+
+class PickleProtocol(object):
+
+	def data_formate_function(self, metric: str, value: int, timestamp: int):
+		"""
+			@return: required data formate when sending data through 'pickle' protocol
+			@return_type: Tuple
```

Comment:
you can use python 3 typing here:

https://docs.python.org/3/library/typing.html#typing.Tuple

```
timestamp: int) -> Tuple[str, Tuple[int, int]]
```